lgc/interface/lgc/Pipeline.h

```diff
@@ -478,6 +478,10 @@ struct GeometryShaderMode {
 struct FragmentShaderMode {
   unsigned pixelCenterInteger;
   unsigned earlyFragmentTests;
+  unsigned depthUnchanged;
+  unsigned depthGreater;
```

Comment:
Why are these an unsigned each, when they appear to be used as flags, and mutually exclusive ones at that? And two of them are unused.

Could you please change to a single entry that is an enum for the conservativeZExport that you want? Ensure that the default setting (presumably "any Z")  is value 0. And remove the unused ones, until you also have the code to use them in LGC.

Reply:
Sure, I've already changed it in the newest pull request.